mldsa/fips202/fips202.c

```diff
@@ -46,13 +52,21 @@ static uint64_t load64(const uint8_t x[8])
  * Arguments:   - uint8_t *x: pointer to the output byte array (allocated)
  *              - uint64_t u: input 64-bit unsigned integer
  **************************************************/
-static void store64(uint8_t x[8], uint64_t u)
+static void store64(uint8_t x[8], const uint64_t u)
```

Comment:
`const` is not too common for arguments passed by value, not by reference. The qualification only restricts what the implementation can do, but it has no impact on the caller.